example/src/Tab/TabPage.tsx

```diff
@@ -0,0 +1,40 @@
+import React, { useState } from 'react';
+import { View } from 'react-native';
+import { Screen, Tab, useTheme } from 'smartway-react-native-ui';
+
+export const TabPage = () => {
+    const theme = useTheme();
+    const [activeTab, setActive] = useState('Zéro-Gâchis');
+
+    const handleTabPress = (tab: string) => {
+        setActive(tab);
+    };
+
+    return (
+        <Screen style={{ backgroundColor: theme.sw.colors.neutral[50] }}>
```

Comment:
Not related to this PR, but I'm pretty sure we always override the default screen color to use `theme.sw.colors.neutral[50]`; I think it could be good to change the default Screen background color to theme.sw.colors.neutral[50] and override it only when necessary (in the 'Enrollement screen and in the 'Synchronization' screen')

Reply:
Yes! I will make a pr for it later, to include it to release!